src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -485,31 +485,35 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
     private Locomotive performCornerSwipe(ScreenCorner corner, SwipeElementDirection direction, float percentage, int duration) {
         Dimension screen = driver.manage().window().getSize();
 
-        int MARGINPX_10 = 10;
+        int X_MIN_MARGIN = 10;
```

Comment:
constants are declared `static final` so they can never change. I would also replace all values of `10` with a single constant.
so `static final int MARGIN = 10` at the top of the class and then
```
case BOTTOM_LEFT:
    from = new Point (MARGIN, screen.getHeight() - MARGIN)`
    break;
```
so you have a concise switch statement and store the margin in only 1 place.